File: 26.remove-duplicates-from-sorted-array.java
/*
 * @lc app=leetcode id=26 lang=java
 *
 * [26] Remove Duplicates from Sorted Array
 */

// @lc code=start
class Solution {
    public int removeDuplicates(int[] nums) {
        int left =0, right = 0;
        for (right = 0; right < nums.length; right++) {
            if(nums[left] == nums[right]) {continue;}
            else{
                nums[++left] = nums[right];
            }
        }
        return left + 1;
        
    }
}
// @lc code=end

File: 26.remove-duplicates-from-sorted-array.js
/*
 * @lc app=leetcode id=26 lang=javascript
 *
 * [26] Remove Duplicates from Sorted Array
 */

// @lc code=start
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    var cur =0;
    for(let i = 1; i<nums.length; i++){
        if (nums[cur] === nums[i]){
            continue;
        }else{
            cur +=1;
            nums[cur] = nums[i];
        }
    }
    return cur+1;
    
};
// @lc code=end

